go.mod

```diff
@@ -3,21 +3,21 @@ module github.com/rhobs/multicluster-observability-addon
 go 1.20
 
 require (
-	github.com/imdario/mergo v0.3.12
+	github.com/cert-manager/cert-manager v1.13.3
 	github.com/openshift/cluster-logging-operator v0.0.0-20231130135759-9270994dc4bc
 	github.com/operator-framework/api v0.17.7
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.3
+	github.com/stretchr/testify v1.8.4
```

Comment:
This is debatable, but a debate for later. I am pretty much sure that metrics folks like to use github.com/efficientgo/core :D (cc @saswatamcode )